### ISSUE TITLE
Create the monkey language interpreter / compiler

### ISSUE THREAD

**Issue by buttercubz** [littledivy/monkeylang#1]
with the introduction of deno 1.6 I think it would be a good idea to use deno compile to create the monkeylang interpreter

**Comment by littledivy** [comment]
I agree, we could setup a CI workflow for producing build artifacts and uploading to Releases.
